packages/hooks/src/useAccountPlatformDetails.ts

```diff
@@ -0,0 +1,10 @@
+import { useStore } from '@deriv/stores';
+
+const useAccountPlatformDetails = () => {
```

Comment:
@hamid-deriv Just a question, Should we rename it to something like `useCurrentAccountDetails` or I'm I misunderstanding the whole thing 😅

Reply:
It can be that. Will update the code.